uwb/example/lib/host_widgets/error_message.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+/// The [ErrorMessage] class.
+///
+/// Shows an error message when the decice isn't compatible with UWB.
+class ErrorMessage extends StatelessWidget {
+  /// The [ErrorMessage] constructor.
+  const ErrorMessage(
+      {Key? key, required this.x, required this.y, required this.error})
+      : super(key: key);
+
+  /// Variable for x-axis of the error message.
+  final double x;
+
+  /// Variable for y-axis of the error message.
+  final double y;
+
+  /// Variable that contains the error message.
+  final String? error;
```

Comment:
Since the `error` is a required parameter of the constructor, it should never be `null`:

```suggestion
  final String error;
```